README.md

```diff
@@ -6,6 +6,13 @@
 
 ## News
 
+Argo Project is moving from CLA to DCO as a more contributor friendly way to track contributions.
+* https://github.com/apps/dco/
+Please start signing off your contributions by doing ONE of the following:
+* Run `git config commit.gpgsign true` in your argo repo to automatically signoff future commits.
```

Comment:
this did not work for me, I think I have an older Git which does not have this setting

Reply:
Whoops, I need to delete the gpgsign part.

Reply:
I deleted the gpgsign part.